spec/system/admin/editing_an_estimate_spec.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe(
+  "Editing an estimate", :js,
+  feature_setting: { user_based_price_groups: true },
+) do
+  let(:facility) { create(:setup_facility) }
+  let(:director) { create(:user, :facility_director, facility:) }
+  let!(:user) { create(:user) }
+  let(:price_group) { user.price_groups.first }
```

Comment:
Because of the new price groups solution, we shouldn't need the feature on, so we should do this: `let(:price_group) { facility.price_groups.first }`